src/Data/Project.hs

```diff
@@ -43,18 +41,6 @@ newtype ProjectException
   = FileNotFound FilePath
     deriving (Show, Eq, Typeable, Exception)
 
-readFile :: (Member (Error SomeException) sig, Carrier sig m)
-         => Project
-         -> File
-         -> m (Maybe Blob)
-readFile Project{..} f =
-  let p         = filePath f
-      candidate = find (\b -> blobPath b == p) projectBlobs
-  in if
-    | p == "/dev/null"  -> pure Nothing
-    | isJust candidate  -> pure candidate
-    | otherwise         -> throwError (SomeException (FileNotFound p))
```

Comment:
This appears to have been the last `MultiWayIf` in this file, so we can turn off the extension.